awkward-cpp/pyproject.toml

```
@@ -0,0 +1,63 @@
+[build-system]
+requires = [
+	"scikit-build-core @ git+https://github.com/agoose77/scikit-build-core@27a60e553733ecffa7b411fe2760152555eaeefa",
```

```suggestion
    "scikit-build-core[pyproject]",
```

The pyproject is optional, but don't think there's a reason not to have it, and it could speed up the build slightly.